test-helpers/src/test/scala/neotypes/BaseSpec.scala

```diff
@@ -0,0 +1,20 @@
+package neotypes
```

Comment:
Why did this have to be moved outside of `core-tests`?

Reply:
https://github.com/neotypes/neotypes/pull/666#discussion_r1306543528

Reply:
@i10416 so if I understand correctly, this will also be removed once everything cross-compiles?

Reply:
Yes.